superset-frontend/src/components/Badge/Badge.stories.tsx

```diff
@@ -73,32 +73,32 @@ InteractiveBadge.argTypes = {
   status: {
     control: {
       type: 'select',
-      options: [undefined, ...STATUSES],
     },
+    options: [undefined, ...STATUSES],
   },
   size: {
     control: {
       type: 'select',
-      options: SIZES.options,
     },
+    options: SIZES.options,
   },
   color: {
     control: {
       type: 'select',
-      options: [undefined, ...COLORS.options],
     },
+    options: [undefined, ...COLORS.options],
   },
   textColor: {
     control: {
       type: 'select',
-      options: [undefined, ...COLORS.options],
     },
+    options: [undefined, ...COLORS.options],
   },
   count: {
     control: {
       type: 'select',
-      options: [undefined, ...Array(100).keys()],
     },
+    options: [undefined, ...Array(100).keys()],
```

Comment:
I'm not even sure what this control does :P 

I think for the most part, all your changes in this PR are just fine... I'm simply trying things out and leaving notes in case you have time/interest to keep messing with all this. 

In this story, the text size one doesn't seem to have any effect. All the rest of the controls work fine, but I'm not sure why the badge is displaying more like a bullet than as a "badge" as it is in the "Badge Gallery" story.

Reply:
I am not sure why that happens. This was the component that made me write this PR. Some props override others, this should at least help with debugging that.